R/cli-wrapper.R

```diff
@@ -38,45 +42,49 @@ share_dat <- function (dir = '.') {
 #'
 #' @export
 #'
-#' @examples \dontrun{
-#'   clone_dat(link = 'dat://datr-chris.hashbase.io', dir = tempdir())
-#' }
+#' @examples
+#' clone_dat(link = 'dat://datr-chris.hashbase.io', dir = tempdir())
 
 clone_dat <- function (link, dir) {
-  if (!dir.exists(dir)) {
-    dir.create(dir)
+
+  if (dat_is_installed(quiet = FALSE)) {
+    if (!dir.exists(dir)) {
```

Comment:
The dir is automatically created by the `dat` cli, this can be removed. What do you think?